config/initializers/permission_override.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'symmetric-encryption'
+
+module KeystoreOverride
```

Comment:
Can you add a comment explaining why it's needed, so we don't forget?

Reply:
Yes, I will do it.